pull_dataset.py

```diff
@@ -10,6 +10,13 @@
 log = logging.getLogger(__name__)
 
 
+def _check_credentials_exist():
+    assert os.environ.get('AWS_ACCESS_KEY_ID') is not None, 'Missing access key'
```

Comment:
Maybe improve those error messages a bit? Users might see them and we can actually tell them more, like which env variable specifically is not set.
Same goes for the one below.

Reply:
Good point, hope this looks better https://github.com/jina-ai/examples/pull/538/commits/43ab7fb24b2ed12033dc77c74c45d8ed5d8eaacf